src/Components/List/List.jsx

```diff
@@ -2,21 +2,54 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../../lib/firebase.js';
 import { formatString } from '../../lib/helpers.js';
+import calculateEstimate from '../../lib/estimates';
 import dayjs from 'dayjs';
 import './List.css';
 
+const getNumberOfPurchases = (item) => {
+  if (item.numberOfPurchases === undefined) {
+    return 1;
+  } else {
+    return item.numberOfPurchases + 1;
+  }
+};
```

Comment:
I like that you've pulled these functions out and given them a clear name! It definitely makes the `purchaseItem` function more readable.